.vscode/tasks.json

```diff
@@ -5,7 +5,17 @@
       "label": "Bazel Build (dbg)",
       "type": "shell",
       "command": "bazel",
-      "args": ["build", "-c", "dbg", "//src/workerd/server:workerd"],
+      "linux": {
+        "args": ["build", "-c", "dbg", "//src/workerd/server:workerd"],
+      },
+      "osx": {
+        // OS X needs additional build options for symbols to work correctly
```

Comment:
A couple of variations to consider:
* `debug_prefix_map_pwd_is_dot` (https://github.com/bazelbuild/bazel/commit/3f46dd0f6d5a042fc28d265411a6014f666a40c1)
* config selection (https://github.com/cloudflare/workerd/pull/537/files).

Reply:
`debug_prefix_map_pwd_is_dot` should be turned on by default. https://blog.bazel.build/2021/01/19/bazel-4-0.html, interesting... :-)

Reply:
Not seeing an alternative to the local spawn strategy.